src/editor/index.php

```diff
@@ -118,7 +132,8 @@ function simplifyPath( $path ) {
 }
 
 // .. as a fully qualified filename
-$strSchemaFQFilename = realpath( simplifyPath( $strConfigPath . ($isDemo?'demo/':'') . $strSchemaFilename) );
+// $strSchemaFQFilename = realpath( simplifyPath( $strConfigPath . ($isDemo?'demo/':'') . $strSchemaFilename) );
+$strSchemaFQFilename = realpath($strConfigPath . ($isDemo?'demo/':'') . $strSchemaFilename );
```

Comment:
Why is the simplifyPath missing here? This might cause problems when the config files are located externally (e.g. /etc/cometvisu/ and only linked into the web root)